src/redux/categories/Categories.js

```diff
@@ -0,0 +1,18 @@
+const CATEGORY = 'CATEGORY';
+const initialCategoryState = [];
+
+const categoryReducer = (state = initialCategoryState, action) => {
+  switch (action.type) {
+    case CATEGORY:
+      return 'Null';
```

Comment:
- [ ] (OPTIONAL) This case should return `UNDER CONSTRUCTION` and not `NULL`. Kindly rectify that too 👍 
![Annotation 2022-06-07 050503](https://user-images.githubusercontent.com/85113567/172293696-5136797a-3da8-4c43-ab9b-e204a8179d75.jpg)

Reply:
Thank you very much, I've applied all of your suggestions.